internal/config/analytics.go

```diff
@@ -0,0 +1,61 @@
+package config
+
+import (
+	"errors"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/spf13/viper"
+)
+
+// AnalyticsConfig defines the configuration for various mechanisms for
+// reporting and querying analytical data for Flipt.
+type AnalyticsConfig struct {
+	Enabled    bool             `json:"enabled,omitempty" mapstructure:"enabled" yaml:"enabled,omitempty"`
```

Comment:
do we need this? I asked previously, but if no analytics backends are configured I think we can not require this as an explicit config value?

Reply:
@markphelps Hmm yeah you are right. We can imply if it is enabled if any of the backends are enabled basically.